corva/app/stream.py

```diff
@@ -59,7 +58,7 @@ def pre_process(self, context: StreamContext) -> None:
         context.event = event
 
     def post_process(self, context: StreamContext) -> None:
-        all_records: List[Record] = list(chain(*[subdata.records for subdata in context.event]))
+        all_records: List[StreamEventData.Record] = list(chain(*[subdata.records for subdata in context.event]))
```

Comment:
Should this be in pre_process?

Reply:
I think `last_processed_timestamp` and `last_processed_depth` should be extracted and saved in `post_process`. so in case stream task fails in `process`, wrong data is not saved to redis and failed task can be restarted with previous values

Reply:
also this class won't be used anymore